src/ansys/geometry/core/plotting/plotter.py

```diff
@@ -389,4 +389,6 @@ def show(
             Keyword arguments for the plotter. Arguments depend of the backend implementation
             you are using.
         """
-        self._backend.show(object=plotting_object, screenshot=screenshot, **plotting_options)
+        if plotting_object is not None:
+            self.plot(plotting_object, **plotting_options)
+        self._backend.show(screenshot=screenshot, **plotting_options)
```

Comment:
I'm curious... why is this needed?

Reply:
Not needed really, this allows the following behaviour:

```python
mesh = pv.Cylinder()
pl = Plotter()
pl.show(mesh)
```

Otherwise, you can only do the following:

```python
mesh = pv.Cylinder()
pl = Plotter()
pl.plot(mesh)
pl.show()
```

I prefer to give more options to the user

Reply:
Oh, gotcha - works for me